src/index.tsx

```diff
@@ -3,7 +3,6 @@ import ReactDOM from 'react-dom';
 import './index.css';
 import App from './App';
 import * as serviceWorker from './serviceWorker';
-import { BrowserRouter, Route, Switch } from "react-router-dom"
```

Comment:
same - aren't we using this?

Reply:
The PR deploy test failed when I had warnings in the code, such as "defined but never used", so i have to delete this never used line.